51-android.rules

```diff
@@ -24,87 +24,87 @@ LABEL="android_usb_rules_begin"
 # Acer
 ATTR{idVendor}!="0502", GOTO="not_Acer"
 #   Iconia Tab A1-830
-ATTR{idProduct}=="3604", ENV{adb_adbfast}="yes"
+ATTR{idProduct}=="3604", GOTO="go_adbfast"
```

Comment:
labels without go_ prefix would be better imho

Reply:
It makes it more compact - I agree too.